frontend/src/stock/AddStockItemModal.tsx

```diff
@@ -49,16 +49,18 @@ function AddItemModal(props: ModalProps) {
         <Modal
             isOpen={props.modalIsOpen}
             contentLabel="Add Modal"
+            ariaHideApp={false}
+            onRequestClose={props.closeModal}
         >
             <CloseIcon closeModal={props.closeModal}/>
             <section>
                 <h2>Neue Position dem Lager hinzufügen</h2>
                 <form onSubmit={handleSubmit}>
 
-                    <div className={"field-label-group"}>
+                    <FieldLabelGroup>
                         <label htmlFor={"name"}>Name/Bezeichnung</label>
                         <input onChange={handleChange} required type={"text"} id={"name"} name={"name"}/>
-                    </div>
+                    </FieldLabelGroup>
```

Comment:
maybe instead of pass label and input as children I would consider include them directly inside FieldLabelGroup and pass needed props  handleChange, name ... .
but this is optional.